src/template.html

```diff
@@ -1,9 +1,9 @@
-<div ng-if="isInEdit" id="qtcOverlay">
+<div ng-if="isInEdit" id="qtcOverlay" style="display: block">
```

Comment:
Is there any specific reason why this was added as inline styling or could this be moved to a css file instead?